catalog/resource_storage_credential.go

```diff
@@ -130,11 +130,6 @@ func ResourceStorageCredential() common.Resource {
 			common.DataToStructPointer(d, storageCredentialSchema, &update)
 			update.Name = d.Id()
 			update.Force = force
-			// We need to set them to empty since these fields are computed
-			if _, ok := d.GetOk("aws_iam_role"); ok {
-				update.AwsIamRole.UnityCatalogIamArn = ""
-				update.AwsIamRole.ExternalId = ""
-			}
```

Comment:
Hmmm, @nkvuong - what do you think, can we remove this?

Reply:
It seems like these two are already marked as `output only` in the current doc, do we still need to make any change?

Reply:
we need to remove them from the docs.

Reply:
Removed, PTAL